tests/rules/assertions/enforceSuppressionCode.js

```diff
@@ -0,0 +1,31 @@
+export default {
```

Comment:
It would be nice to have here some examples of block comments as well, see: https://flow.org/en/docs/errors/

Reply:
Cool found issue with block comments so fixed it